Containerfile.rpmbuild

```diff
@@ -0,0 +1,10 @@
+FROM registry.centos.org/centos
+WORKDIR $APP_ROOT/src
+RUN yum -y install git-core rpm-build dnf-plugins-core 'dnf-command(builddep)'
+COPY . .
+RUN mkdir -p ~/rpmbuild/SOURCES/ && \
```

Comment:
`rpmdev-setuptree` ?

Reply:
This is part of `rpmdevtools`, which uses 40MB of additional disk space because of its deps. `mkdir` should be good enough?